masq_lib/src/shared_schema.rs

```diff
@@ -48,6 +48,18 @@ pub const EARNING_WALLET_HELP: &str =
      (case-insensitive). If you already have a derivation-path earning wallet, don't supply this. \
      If you have supplied an earning wallet address before, either don't supply it again or be \
      careful to supply exactly the same one you supplied before.";
+pub const EXIT_LOCATION_HELP: &str =
+    "Choose your Exit Location for access the internet. You can choose from all countries, available in \
+    your Neighborhood. \n\
+    You can set exit-location --country-codes \"CZ\" without --fallback-routing argument and in that case, \
+    the fallback-routing will be set OFF. In case, you do not specify the --country-codes and neither --fallback-routing, \
+    then country-codes will be set OFF and fallback-routing will be set ON.\n\n\
+        exit-location --country-codes --fallback-routing                    disable exit-location\
+        exit-location --fallback-routing                                    disable exit-location\
+        exit-location                                                       disable exit-location\n\n\
+        exit-location --country-codes \"CZ,PL|SK\" --fallback-routing       fallback-routing is ON \
+        exit-location --country-codes \"CZ|SK\"                             fallback-routing is OFF\
+    ";
```

Comment:
Maybe a `masq> ` at the beginning of the line and a `//` before the comment? (Of course, many of our users may not understand `//`; hopefully they'll get it from context.